cli/internal/packagemanager/packagemanager_test.go

```diff
@@ -100,37 +101,39 @@ func TestParsePackageManagerString(t *testing.T) {
 }
 
 func TestGetPackageManager(t *testing.T) {
+	cwd, err := fs.GetCwd()
```

Comment:
These changes make it look like we're testing something else. It might actually bring these in line with the application code based on my review, but I wanted to make sure that's the case.

Reply:
This particular test doesn't use that value either way. All of the sample `PackageJSON` instances have the package manager specified already, so `readPackageManager` always returns a result and we never get to `detectPackageManager`.

Reply:
👍